code/src/pages/MovieDetails.js

```diff
@@ -0,0 +1,52 @@
+import React, { useState, useEffect } from "react";
+import { useParams } from "react-router-dom";
+
+import "./moviedetails.css";
+
+export const MovieDetails = () => {
```

Comment:
We think this component is too good to be true! So clean and tidy that Mari-Condo would be so proud of you! 🤓